astropy/time/formats.py

```diff
@@ -487,7 +516,7 @@ def to_value(self, **kwargs):
         dt_end = (jd1_end - jd1_start) + (jd2_end - jd2_start)
         decimalyear = iy_start + dt / dt_end
 
-        return super().to_value(jd1=decimalyear, jd2=0., **kwargs)
+        return super().to_value(jd1=decimalyear, jd2=np.float64(0.0), **kwargs)
```

Comment:
Why this change? (Out of curiosity, doesn't matter too much)

Reply:
There are a couple places, e.g. `utils.twoval_to_longdouble` that use `val2.astype(..)`.  A standard Python float does not have `.astype()` and this crashes.  The original testing did not cover this case.

Reply:
If we want to ensure things are always arrays (probably good, since they can be `.view`d, etc.), maybe just do `np.array(0.)`?

Reply:
Does it matter?  Both cases in question are just `val2=..` args in `to_value()` which combine `val1` and `val2` into the single "value".  The `val2` value is lost in that process.

Reply:
OK, good point. Let's keep it as you have it.